src/components/Monaco/DiffEditor.tsx

```diff
@@ -0,0 +1,39 @@
+import { Editor, DiffEditor as MonacoDiffEditor } from '@monaco-editor/react'
+import { editor } from 'monaco-editor'
+import { ComponentProps } from 'react'
+
+import { useTheme } from '@/hooks/useTheme'
+
+import { DEFAULT_OPTIONS } from './defaultOptions'
+const DEFAULT_DIFF_OPTIONS: editor.IDiffEditorOptions = {
+  ...DEFAULT_OPTIONS,
+  readOnly: true,
+  renderOverviewRuler: false,
+  lineNumbers: 'off',
+}
+
+interface Props extends ComponentProps<typeof Editor> {
+  original?: string
+}
+
+export function DiffEditor({ original, value, language, options }: Props) {
+  const theme = useTheme()
+  return (
+    <MonacoDiffEditor
+      options={{
```

Comment:
Is there any way to disable this popover?
<img width="299" alt="image" src="https://github.com/user-attachments/assets/7464e053-fd60-4b32-89f4-2f14f04b86e8" />

Reply:
There's a [bug](https://github.com/microsoft/monaco-editor/issues/3873) in monaco which doesn't allow disabling it with options, added CSS rule to hide it